### ISSUE TITLE
custom headers added in console do not accept free text name

### ISSUE THREAD

**Issue by carobazan** [mulesoft/api-console#505]
Create a raml file with this code:
```
#%RAML 1.0
title: test API
version: v1
/users:
    displayName: User
    get:
      responses:
        200:
          description: |
            OK
```
Go to console, and navigate to Try it for /users GET method, click on Headers tab and add click on add custom header

The text box does not allow free text to give a header a name, click does inside the text box header name does not seem to have any effect, nevertheless when retrying it displays a list of available values but in erratic manner 
It is not clear that the header needs to be selected from a list of values

**Comment by jarrodek** [comment]
Let me rephrase this.
Do you mean that the suggestions overlay covers the input field?
